pkg/commands/artifact/run.go

```diff
@@ -233,6 +236,19 @@ func scan(ctx context.Context, opt Option, initializeScanner InitializeScanner,
 		}
 	}
 
+	// Do not load config file for secret scanning
+	if !slices.Contains(opt.SecurityChecks, types.SecurityCheckSecret) {
+		opt.SecretConfigPath = ""
+	} else {
+		ver := fmt.Sprintf("v%s", opt.AppVersion)
```

Comment:
you use `ver` only for links to the doc, right?

Reply:
maybe `latest` is easier. what do you think?

Reply:
If the old Trivy refers to the latest document, some of feature may be missing. Let's say we add `skip_large_files: true` in v0.28.0, but v0.27.0 doesn't have it. It will confuse users.

Reply:
>you use ver only for link to the doc, right?

Right.

Reply:
i meant that now this link will work for all version from v0.27.0, and won't work for v0.26.0 or earlier. It looks like `latest`.
but if we change something and the link can be broken.

you're right. i agree

Reply:
yes, if we create a log message instead of `ver`, it'll create a string once instead of twice, right?
but it doesn't matter in our case )